packages/grafana-ui/src/components/DrilldownLinks/DrilldownLinkEditor.tsx

```diff
@@ -1,51 +1,15 @@
-// Libraries
-import React, { FC, ChangeEvent, useState } from 'react';
+import React, { FC, useState, ChangeEvent } from 'react';
+import { PanelDrillDownLink } from '../../index';
+import { FormField, Switch } from '../index';
 
-// Components
-import { PanelOptionsGroup, PanelDrillDownLink, FormField, Switch } from '@grafana/ui';
-
-export interface Props {
-  value: PanelDrillDownLink[];
-  onChange: (links: PanelDrillDownLink[]) => void;
-}
-
-export const PanelLinksEditor: FC<Props> = React.memo(({ value, onChange }) => {
-  const onAdd = () => {
-    onChange([...value, { url: '', title: '' }]);
-  };
-
-  const onLinkChanged = (linkIndex: number, newLink: PanelDrillDownLink) => {
-    onChange(
-      value.map((item, listIndex) => {
-        if (linkIndex === listIndex) {
-          return newLink;
-        }
-        return item;
-      })
-    );
-  };
-
-  const onRemove = (link: PanelDrillDownLink) => {
-    onChange(value.filter(item => item !== link));
-  };
-
-  return (
-    <PanelOptionsGroup title="Drilldown links" onAdd={onAdd}>
-      {value.map((link, index) => (
-        <LinkEditor key={index.toString()} index={index} value={link} onChange={onLinkChanged} onRemove={onRemove} />
-      ))}
-    </PanelOptionsGroup>
-  );
-});
-
-export interface LinkEditor {
+export interface DrilldownLinkEditor {
```

Comment:
Think this should be suffixed Props (Think you had the same comment on my PR)

Reply:
or now that it's in it's own file just Props :)

Reply:
Heh, yeah, exactly.